tests/dummy/app/templates/button.hbs

```diff
@@ -8,12 +8,13 @@
 {{/paper-toolbar}}
 {{#paper-content class="md-padding demo-buttons"}}
 <div class="doc-content">
-  <p>
+  <div layout="row" flex>
```

Comment:
OK, but let's change the whole addon at once. I'm pretty sure I copied this from another component. Horizontal versus vertical updating, one might say.